code/modules/shuttle/shuttles/ert.dm

```diff
@@ -88,10 +106,77 @@
 	preferred_direction = SOUTH
 	port_direction = NORTH
 
+// ERT Shuttle 4
+/obj/docking_port/mobile/emergency_response/small
+	name = "Rescue Shuttle"
+	id = MOBILE_SHUTTLE_ID_ERT_SMALL
+	preferred_direction = SOUTH
+	port_direction = NORTH
+	width = 6
+	height = 9
+	var/port_door
+	var/starboard_door
+
+/obj/docking_port/mobile/emergency_response/small/Initialize(mapload)
+	. = ..()
+	external_doors = list()
+	for(var/place in shuttle_areas)
+		for(var/obj/structure/machinery/door/air in place)
+			if(air.id == "starboard_door")
```

Comment:
are these ids set as instances on the maps, btw?

if so, we could always subtype + define them for a bit of extra clarity

Reply:
These are currently set in the map editor, they could be subtyped but I thought that this way would allow for any door type to be used and it be left at the mappers discretion

Reply:
fair enough!